docs/lib/datastore/fragments/flutter/datastore-events.md

```diff
@@ -0,0 +1,23 @@
+```dart
+AmplifyDataStore.events.listenToDataStore((event) {
+    if (event["eventName"] == "networkStatus") {
+        setState(() {
+            networkIsUp = event["active"];
+        });
+    }
+});
+```
+
+<amplify-callout>
+
+Please note that all amplify-flutter DataStore Hub events are currently structured as serialized maps.  The event name is specified in an 'eventName' attribute within the map.
```

Comment:
Edit for conciseness & clarity. 
"All Flutter DataStore Hub events are currently structured as serialized maps."

Reply:
changed. Still kept "The event name is specified in an 'eventName' attribute within the map." to make it easier to read the events. This will be changed soon with Event types in dart